app/uk.gov.hmrc.helptosavestridefrontend/views/helpers/input_text.scala.html

```diff
@@ -21,11 +21,12 @@
 inputClass: Option[String] = None,
 hint: Option[String] = None,
 name: String = "value",
+containerClass: Option[String] = None,
 labelClass: Option[String] = None,
 errorMessage: Option[Html] = None
 )(implicit messages: Messages)
 
-<div class="form-group @if(errorMessage.isDefined){form-group-error}">
+<div class="form-group @if(errorMessage.isDefined){form-group-error} @if(containerClass.nonEmpty){@containerClass}">
```

Comment:
does the `containerClass` thing actually work here? It looks like you'll end up with `Some(blah blah)` in the resulting HTML. Doing something like `containerClass.getOrElse("")` seems to make more sense here

Reply:
This does work for me, the intended class gets written to the element when provided.